### DISCUSSION TITLE
Container with only one side

### DISCUSSION THREAD

**Discussion by htmujahid** [tailwindlabs/tailwindcss#9790] [Ideas]
Hi,
I am not quite sure about this issue, whether it's or not. Sometimes I want to implement design in which content is overflowing from container's right side. I face issue this is not doable in tailwind current class. I have to implement my container class and container-left and container-right class. So I think these classes should be implement by default which will help a lot.
Regards...

**Comment by codeflorist** [comment]
Check out https://github.com/LucidNinja/tailwind-container-break-out

**Reply by htmujahid** [reply]
This is breaking out from both sides. Is not there any way to break from only one side

**Reply by codeflorist** [reply]
It can do that too using `ml-break-out` or `mr-break-out`. Read the docs.
